flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/PipelinedSubpartition.java

```diff
@@ -305,6 +308,10 @@ BufferAndBacklog pollBuffer() {
 		}
 	}
 
+	Buffer buildSliceBuffer(BufferConsumerWithPartialRecordLength buffer) {
+		return buffer.build();
+	}
```

Comment:
This method is going to be @override by PipelinedApproximateSubPartition later.

But I think it might be confused if changing it here. I will inline the change for now.